modules/compliance-operator-hcp-install.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-installation.adoc
+
+:_content-type: PROCEDURE
+[id="installing-compliance-operator-hcp_{context}"]
+= Installing the Compliance Operator on Hosted Control Planes
+
+The Compliance Operator can be installed in Hosted Control Planes using the OperatorHub by creating a Subscription file.
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You must have `admin` privileges.
+
+.Procedure
+
+. Define a `Subscription` object:
++
+.Example `subscription-object.yaml`
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: compliance-operator-
```

Comment:
nit: is it missing a sub in the end?

Reply:
Oh weird - I thought I included that in my pushed changes. I'll update it to `name: compliance-operator-sub` as expected.